webdriver/tests/user_prompts/send_alert_text.py

```diff
@@ -0,0 +1,67 @@
+from tests.support.asserts import assert_error, assert_success
+
+
+def send_alert_text(session, body=None):
```

Comment:
Can you please test various forms of invalid input to the Send Alert Text command?

Reply:
Could you provide me with some examples of invalid input? Thank you

Reply:
Null, undefined, number, object, array

Reply:
fixed